src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -597,4 +599,21 @@ public function removeParameter($parameterName)
             }
         }
     }
+
+    /**
+     * Fetch an array of all return statements found within this function.
```

Comment:
I've not renamed the API but added a doc note. This no longer will return `return` statements found within closures or anon classes.